app/controllers/users_controller.rb

```diff
@@ -9,7 +9,8 @@ def create
     @user = User.new(user_params)
 
     if @user.save
-      redirect_to '/index'
+      redirect_to '/articles'
+      login!
```

Comment:
It might be a good idea to switch the order of these just in case the login fails, the user can't be redirected in that instance.

Reply:
Good catch, changed now